src/wallet/rpcwallet.cpp

```diff
@@ -248,7 +248,7 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
         if (meta->HasKeyOrigin()) {
             ret.pushKV("hdkeypath", meta->key_origin.pathToString());
             ret.pushKV("hdseedid", meta->hd_seed_id.GetHex());
-            ret.pushKV("hdmasterfingerprint", HexStr(meta->key_origin.fingerprint, meta->key_origin.fingerprint + 4));
+            ret.pushKV("hdmasterfingerprint", HexStr(Span<unsigned char>(meta->key_origin.fingerprint, 4)));
```

Comment:
arrays can be directly passed as spans, since they are implicitly converted
https://github.com/PIVX-Project/PIVX/blob/60d36292bc601619b3eb7b60d3e827266cf6566d/src/span.h#L144-L146

So we can just do
```suggestion
            ret.pushKV("hdmasterfingerprint", HexStr(meta->key_origin.fingerprint));
```

Reply:
yep 👍 .